test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -186,7 +187,12 @@ class ExecutionFramework
 	{
 		return encodeArgs(u256(0x20), u256(_arg.size()), _arg);
 	}
-
+	static u256 fixed(dev::bigint _numerator, dev::bigint _denominator, int _fixedBits)
+	{
+		rational _value = rational(dev::bigint(_numerator), dev::bigint(_denominator));
+		rational value = _value * boost::multiprecision::pow(bigint(2), _fixedBits);
```

Comment:
ah, that's what you meant. I mean you could...but I think that would look very messy.